src/test/java/temporary/tests/TestCreateAccountPage.java

```diff
@@ -0,0 +1,34 @@
+package temporary.tests;
+
+import models.AccountModel;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import pages.createAccountPage.CreateAccountPage;
+
+import static steps.BaseSteps.openPage;
+import static steps.CreateAccountPageSteps.fillCreateAccountForm;
+import static steps.CreateAccountPageSteps.goToCreateAccountPage;
+import static steps.CreateAccountPageSteps.submitCreateAccountForm;
+import static steps.CreateAccountPageSteps.validateFirstNameInNavbar;
+
+public class TestCreateAccountPage extends BaseTest {
+    protected static CreateAccountPage createAccountPage;
+
+    @BeforeClass
+    public static void setUp() {
+        createAccountPage = new CreateAccountPage();
+    }
+
+    @Test
+    public void createAccount() {
+        // Given:
```

Comment:
You can add empty line before each // it will logically and visually split the test:)